torchrl/envs/transforms/transforms.py

```diff
@@ -1718,18 +1719,23 @@ def __init__(
 
     def reset(self, tensordict: TensorDictBase) -> TensorDictBase:
         """Resets _buffers."""
-        _reset = tensordict.get(
-            "_reset",
-            None,
-        )
-        if _reset is None:
-            _reset = torch.ones(
-                tensordict.batch_size,
-                dtype=torch.bool,
-                device=tensordict.device
-                if tensordict.device is not None
-                else torch.device("cpu"),
+        _reset = (
+            tensordict.get(
+                "_reset",
+                torch.ones(
```

Comment:
let's not do that. The None has a reason: we don't want to allocate tensors that we don't use. By setting default to None, we can then allocate it only when needed

Reply:
Got it thanks!